controllers/resources/status.go

```diff
@@ -71,6 +71,24 @@ func UpdateStatusNodeHealthy(nodeName string, nhc *remediationv1alpha1.NodeHealt
 	}
 }
 
+func UpdateStatusNodeDelayedHealthy(nodeName string, nhc *remediationv1alpha1.NodeHealthCheck, remediationCRs []unstructured.Unstructured) {
+	for i, _ := range nhc.Status.UnhealthyNodes {
+		if isNodeHealthyDelayed(nodeName, remediationCRs) {
+			nhc.Status.UnhealthyNodes[i].HealthyDelayed = true
+		}
```

Comment:
Doesn't this set `.HealthyDelayed = true` for _any_ `UnhealthyNode` even if only the `nodeName` is delayed?